coffee-dto/coffee-dto-xsd/src/main/resources/xsd/hu/icellmobilsoft/coffee/dto/common/common.xsd

```diff
@@ -321,12 +321,12 @@
 	</xsd:simpleType>
 	<xsd:simpleType name="TimestampType">
 		<xsd:annotation>
-			<xsd:documentation>Időbélyeg típus ISO "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
-				formatumhoz</xsd:documentation>
+			<xsd:documentation>Időbélyeg típus ISO "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ"
+				formátumhoz</xsd:documentation>
 		</xsd:annotation>
 		<xsd:restriction base="xsd:dateTime">
 			<xsd:pattern
-				value="\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(.\d{1,3})?Z" />
+				value="\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(.\d{1,6})?Z" />
```

Comment:
volt meg egy regi hibank a milisecnel, a helyes pattern `\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d{1,6})?Z`

Reply:
migrációs doksiba is vedd fel légyszíves a pattern változást